src/loaders/core_text.rs

```diff
@@ -1023,7 +1045,7 @@ pub(crate) fn piecewise_linear_find_index(query_value: f32, mapping: &[f32]) ->
         Ok(index) => return index as f32,
         Err(upper_index) => upper_index,
     };
-    if upper_index == 0 || upper_index >= mapping.len() {
+    if upper_index == 0 {
```

Comment:
Mind reverting this?

Reply:
done.  this is probably why the rebase got messed up.  not sure how it happened